### PR TITLE
Web Cloud - Email migration - information about contacts

### PR DESCRIPTION
**Pull request by Kazarus4**
Quifix realised for all subs after Techlead request.

JIRA created => SK984 => need peer review for FR and all subs versions.

### CODE REVIEW

File: pages/web/microsoft-collaborative-solutions/migration_control_panel/guide.fr-fr.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Pour cela, connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/
 
 > [!primary]
 >
-> Votre compte OVHcloud doit préalablement être contact administrateur et contact technique du service E-mail Pro ou Exchange vers lequel vous migrez.
+> Votre compte OVHcloud doit préalablement être contact *administrateur* **et** contact *technique* du service MX plan à migrer **et** du service E-mail Pro ou Exchange vers lequel vous migrez.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/migration_control_panel/guide.fr-fr.md]:
```suggestion
> Votre compte OVHcloud doit préalablement être contact administrateur **et** contact technique du service MX plan à migrer, **ainsi que** du service E-mail Pro ou Exchange vers lequel vous migrez.
```

File: pages/web/microsoft-collaborative-solutions/migration_control_panel/guide.fr-ca.md [hunk 2]
```diff
@@ -54,6 +54,13 @@ Pour cela, connectez-vous à votre [espace client OVHcloud](https://ca.ovh.com/a
 
 #### 3.1 Migrer une offre MXPlan historique <a name="VersionHistoriqueMxplan"></a>
 
+> [!primary]
+>
+> Votre compte OVHcloud doit préalablement être contact administrateur **et** contact technique du service MX plan à migrer, **ainsi que** du service E-mail Pro ou Exchange vers lequel vous migrez.
+>
+> Pour plus d'information sur les changements de contacts, consultez notre guide pour [gérer les contacts de ses services](../../customer/gestion-des-contacts/).
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/migration_control_panel/guide.fr-ca.md]: Please replace the link with an absolute one.

Comment (OVH-Team-Guides) [reply pages/web/microsoft-collaborative-solutions/migration_control_panel/guide.fr-ca.md]: Same for fr-fr

Comment (Kazarus4) [reply pages/web/microsoft-collaborative-solutions/migration_control_panel/guide.fr-ca.md]: Done

**Pull request merged**